pkg/menu/menu.go

```diff
@@ -84,15 +84,20 @@ func processInput(introwords string, location int, wid int, ht int, isValid vali
 				warning.Text = ""
 				ui.Render(warning)
 			}
-			input.Text += k
+			if k == "<Space>" {
+				input.Text += " "
+			} else if k[0:1] != "<" {
```

Comment:
I'm a little confused, why do we need this check?

Reply:
because for example we hit left arrow, the termui's ui.PollEvents() will push a string"<Left>" to the channal. and all other special key like arrow, F1,etc on the keyboard will be treat in this way. So if the key is not begin at '<' it means it's a alphabet or number or .

Reply:
ok, will do